internal/env/env.go

```diff
@@ -0,0 +1,37 @@
+package env
+
+const (
+	DevboxCache              = "DEVBOX_CACHE"
+	DevboxCLICloudShell      = "DEVBOX_CLI_CLOUD_SHELL"
+	DevboxDebug              = "DEVBOX_DEBUG"
+	DevboxFeaturePrefix      = "DEVBOX_FEATURE_"
+	DevboxGateway            = "DEVBOX_GATEWAY"
+	DevboxDoNotUpgradeConfig = "DEVBOX_DONT_UPGRADE_CONFIG"
+	DevboxRegion             = "DEVBOX_REGION"
+	DevboxSearchEndpoint     = "DEVBOX_SEARCH_ENDPOINT"
+	DevboxShellEnabled       = "DEVBOX_SHELL_ENABLED"
+	DevboxShellStartTime     = "DEVBOX_SHELL_START_TIME"
+	DevboxVM                 = "DEVBOX_VM"
+
+	DoNotTrack = "DO_NOT_TRACK"
+
+	LauncherVersion = "LAUNCHER_VERSION"
+
+	SSHTTY = "SSH_TTY"
+
+	StartWebTerminal = "START_WEB_TERMINAL"
+
+	XDGDataHome   = "XDG_DATA_HOME"
+	XDGConfigHome = "XDG_CONFIG_HOME"
+	XDGCacheHome  = "XDG_CACHE_HOME"
+	XDGStateHome  = "XDG_STATE_HOME"
+)
```

Comment:
Many of these don't need to be exported, we should use camelCase for those so we can use nicer names for the exported functions (e.g. `DoNotTrack()`)

Reply:
Agree. I'll create a PR to unexport some constants and use functions outside.

Reply:
See https://github.com/jetpack-io/devbox/pull/954